allennlp/training/checkpointer.py

```diff
@@ -21,29 +23,71 @@ class Checkpointer(Registrable):
     Dict[str, Any]), but they will be fed to `torch.save` so they should be serializable
     in that sense. They will also be restored as Dict[str, Any], which means the calling
     code is responsible for knowing what to do with them.
+
+    # Parameters
+
+    num_serialized_models_to_keep : `int`, optional (default=20)
+        Number of previous model checkpoints to retain.  Default is to keep 20 checkpoints.
```

Comment:
Why is the default so high anyways? I've never been in a situation where I wanted the model from 19 iterations ago, but I have run out of disk space many times.

Reply:
Yeah, good point.  I'm going to change it to 2.